### PR TITLE
Fix lever to death priest shargon and the ravager

### PR DESCRIPTION
**Pull request by ghost**
# Description

fix lever to Death Priest Shargon and The Ravager

obraz

### CODE REVIEW

File: data/scripts/actions/quests/dark_trails/lever_death_priest_shargon.lua [hunk 1]
```diff
@@ -39,9 +39,9 @@ function leverDeathPriestShargon.onUse(player, item, fromPosition, target, toPos
 
 		local storePlayers, playerTile = {}
 		for i = 1, #setting.playerPositions do
-			local creature = Tile(playerPositions[i]):getTopCreature()
+			local creature = Tile(setting.playerPositions[i]):getTopCreature()
 			if not creature or not creature:isPlayer() then
-				player:sendMessageCancel("You need 5 of players to fight with Death Priest Shargon.")
+				doPlayerSendCancel(player, "You need 5 of players to fight with Death Priest Shargon.")
```
Comment (dudantas) [inline data/scripts/actions/quests/dark_trails/lever_death_priest_shargon.lua]: You can return this?

File: data/scripts/actions/quests/dark_trails/lever_the_ravager.lua [hunk 2]
```diff
@@ -31,9 +31,9 @@ function leverTheRavager.onUse(player, item, fromPosition, target, toPosition, i
 
 		local storePlayers, playerTile = {}
 		for i = 1, #setting.playerPositions do
-			local creature = Tile(playerPositions[i]):getTopCreature()
+			local creature = Tile(setting.playerPositions[i]):getTopCreature()
 			if not creature or not creature:isPlayer() then
-				player:sendMessageCancel("You need 4 of players to fight with The Ravager.")
+				doPlayerSendCancel(player, "You need 4 of players to fight with The Ravager.")
```
Comment (dudantas) [inline data/scripts/actions/quests/dark_trails/lever_the_ravager.lua]: And this

File: data/scripts/actions/quests/dark_trails/lever_death_priest_shargon.lua (hunk 1 above)
Comment (ghost) [reply data/scripts/actions/quests/dark_trails/lever_death_priest_shargon.lua]: yes i can return it by clicking the lever and player < 5

File: data/scripts/actions/quests/dark_trails/lever_the_ravager.lua (hunk 2 above)
Comment (ghost) [reply data/scripts/actions/quests/dark_trails/lever_the_ravager.lua]: yes i can return it by clicking the lever and player < 4

Comment (ghost) [reply data/scripts/actions/quests/dark_trails/lever_the_ravager.lua]: i test always my change

File: data/scripts/actions/quests/dark_trails/lever_death_priest_shargon.lua (hunk 1 above)
Comment (majestyotbr) [reply data/scripts/actions/quests/dark_trails/lever_death_priest_shargon.lua]: He meant to revert the message from `doPlayerSendCancel` (old api) to `player:sendCancelMessage` (updated api).

File: data/scripts/actions/quests/dark_trails/lever_the_ravager.lua (hunk 2 above)
Comment (majestyotbr) [reply data/scripts/actions/quests/dark_trails/lever_the_ravager.lua]: He meant to revert the message from `doPlayerSendCancel` (old api) to `player:sendCancelMessage` (updated api).

**Comment by majestyotbr** [comment]
@rioma1990 Can you do the requested changes? Thanks.

**Comment by ghost** [comment]
> 
> 
> @rioma1990 Can you do the requested changes? Thanks.

corrected player:sendCancelMessage not player:sendMessageCancel  :)

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**